src/cryptenroll/cryptenroll.c

```diff
@@ -118,6 +121,7 @@ static int help(void) {
                "\n%3$sSimple Enrollment:%4$s\n"
                "     --password        Enroll a user-supplied password\n"
```

Comment:
btw, i wonder if we should tweak password enrollment so that if the user types in an empty pw we automatically turn it into a slot marked with "empty" too

Reply:
I suppose we could :shrug:. I didn't because I didn't want to change the behavior in some new and unexpected way